datafusion/physical-plan/src/filter.rs

```diff
@@ -330,8 +330,7 @@ impl DisplayAs for FilterExec {
                 write!(f, "FilterExec: {}{}", self.predicate, display_projections)
             }
             DisplayFormatType::TreeRender => {
-                // TODO: collect info
-                write!(f, "")
+                write!(f, "predicate={}", self.predicate)
```

Comment:
For anyone else following along, this is what it takes to implement the `tree` format for `FilterExec`

The rest of the PR is comments and tests